### DISCUSSION TITLE
How would this "Sentry Notification" app be working?

### DISCUSSION THREAD

**Discussion by mattenz** [timdorr/tesla-api#754] [General]
This app (https://apps.apple.com/app/id6450215698) claims that it can alert you whenever a sentry event happens in the car. Based on what I know of the Tesla API I didn't think this was possible? Yet, it has people claiming that "it works".

I'm obviously not brave enough to install it and give it my Tesla account credentials, but this isn't possible through the API, right?

**Comment by MagoKimbra** [comment]
I installed it on Android a few weeks ago and it actually works. Then I took it away because he wants a few euros a month and it seems too much to me.
It takes 6 to 10 seconds after activation for the notification to arrive, but it arrives.
Actually, from what I know, there is no API command that does this, so I don't understand how it is possible..
